src/pages/api/graphql/index.ts

```diff
@@ -0,0 +1,12 @@
+import { createYoga, createSchema } from 'graphql-yoga'
+import { resolvers } from './resolvers'
+import { typeDefs } from './typeDefs'
+
+const schema = createSchema({
+  typeDefs,
+  resolvers,
+})
+
+export default createYoga({
```

Comment:
Although putting this file in `api` is correct, remember we are using the new `app` folder API instead. This should be moved to a `route.js` file inside the `app` folder. Here is the documentation: https://nextjs.org/docs/app/api-reference/file-conventions/route